RELEASE_NOTES.md

```diff
@@ -5,17 +5,19 @@ _Date TBD_
 
 ### Features Added
 * Improve website performance:
-  * Initialize cache stores asynchronously at server startup
-  * Increase cache timeout to 1 hour
-  * Add caching for /certification_ids API call
-  * Add caching for /pending API call
-  * Add caching for /search API call
-  * Update caching for /search_options API call
-  * Cache underlying data that improves call speed for many other API calls
-* TO-DO DURING DEPLOYMENT: Add these lines to environment.properties:
-  * `enableCacheInitialization=true`
-  * `cacheClearTimeoutSecs=15`
-  * `cacheInitializeTimeoutSecs=300`
+	Initialize cache stores asynchronously at server startup
+	Increase cache timeout to 1 hour
+	Add caching for /certification_ids
+	Add caching for /pending
+	Add caching for /search
+	Add caching for /decertifications/developers
+	Add caching for /decertifications/certified_products
+	Update caching for /search_options API call
+	Cache underlying data that improves call speed for many other API calls
+* TO-DO DURING DEPLOYMENT: Add these lines to environment.properties: 
+	enableCacheInitialization=true
+	cacheClearTimeoutSecs=15
+	cacheInitializeTimeoutSecs=300
```

Comment:
I think Andrew slightly changed the formatting here along with adding backticks around the properties - not sure if/why we need to change it back.